src/ripple/app/paths/impl/Steps.h

```diff
@@ -189,7 +189,16 @@ class Step
     */
     virtual
     bool
-    dry (EitherAmount const& out) const = 0;
+    isZero (EitherAmount const& out) const = 0;
+
+    /**
+       Return true if the step should be considered inactive. A strand may be inactive even if there
+       is more liquidity to be had if a step has consumed too many offers.
+     */
```

Comment:
The second sentence is a little confusing. Consider: _"A strand that has additional liquidity may be marked inactive if a step has consumed too many offers."_

Reply:
Fixed